### ISSUE TITLE
Confused about package status (package broken)

### ISSUE THREAD

**Issue by essenciary** [JuliaDatabases/MySQL.jl#95]
1. the package is broken for me despite running v0.3.0 which according to https://pkg.julialang.org was released 5 days ago. 
2. the README has nothing in common with the methods list returned in the REPL (there's no `.query` and `.Query` but only the "old" `mysql_*` methods). 
3. it internally calls `DataFrames` and crashes despite not listing `DataFrames` as a dependency in `REQUIRE` (I'm running `DataFrames` 0.11)

```julia
genie> MySQL.query(conn, "SHOW COLUMNS from tweets")
ERROR: UndefVarError: query not defined

genie> MySQL.Query(conn, "SHOW COLUMNS from tweets")
ERROR: UndefVarError: Query not defined

genie> MySQL.
@isdefined                             MYSQL_OPT_SSL_CIPHER                    MYSQL_TYPE_DATETIME                     MySQLStatementIterator                  mysql_init_dataframe
CLIENT_MULTI_STATEMENTS                MYSQL_OPT_SSL_CRL                       MYSQL_TYPE_DECIMAL                      NOT_NULL_FLAG                           mysql_insert_id
LIST                                   MYSQL_OPT_SSL_CRLPATH                   MYSQL_TYPE_DOUBLE                       eval                                    mysql_interpret_field
MEM_ROOT                               MYSQL_OPT_SSL_KEY                       MYSQL_TYPE_ENUM                         mysql_affected_rows                     mysql_is_nullable
MYSQL_BIND                             MYSQL_OPT_SSL_VERIFY_SERVER_CERT        MYSQL_TYPE_FLOAT                        mysql_autocommit                        mysql_lib
MYSQL_DATA                             MYSQL_OPT_USE_EMBEDDED_CONNECTION       MYSQL_TYPE_GEOMETRY                     mysql_binary_interpret_field            mysql_load_string_from_resultptr
MYSQL_DATA_FRAME                       MYSQL_OPT_USE_REMOTE_CONNECTION         MYSQL_TYPE_INT24                        mysql_bind_array                        mysql_metadata
MYSQL_DATETIME_FORMAT                  MYSQL_OPT_USE_RESULT                    MYSQL_TYPE_LONG                         mysql_bind_init                         mysql_next_result
MYSQL_DATE_FORMAT                      MYSQL_OPT_WRITE_TIMEOUT                 MYSQL_TYPE_LONGLONG                     mysql_close                             mysql_num_fields
MYSQL_DEFAULT_AUTH                     MYSQL_PLUGIN_DIR                        MYSQL_TYPE_LONG_BLOB                    mysql_connect                           mysql_num_rows
MYSQL_DEFAULT_PORT                     MYSQL_READ_DEFAULT_FILE                 MYSQL_TYPE_MEDIUM_BLOB                  mysql_date                              mysql_options
MYSQL_DEFAULT_SOCKET                   MYSQL_READ_DEFAULT_GROUP                MYSQL_TYPE_NEWDATE                      mysql_datetime                          mysql_query
MYSQL_ENABLE_CLEARTEXT_PLUGIN          MYSQL_REPORT_DATA_TRUNCATION            MYSQL_TYPE_NEWDECIMAL                   mysql_disconnect                        mysql_real_connect
MYSQL_FIELD                            MYSQL_RES                               MYSQL_TYPE_NULL                         mysql_errno                             mysql_real_escape_string
MYSQL_INIT_COMMAND                     MYSQL_ROW                               MYSQL_TYPE_SET                          mysql_error                             mysql_result_to_dataframe
MYSQL_NO_DATA                          MYSQL_ROWS                              MYSQL_TYPE_SHORT                        mysql_escape                            mysql_stmt_affected_rows
MYSQL_OPT_BIND                         MYSQL_SECURE_AUTH                       MYSQL_TYPE_STRING                       mysql_execute                           mysql_stmt_bind_param
MYSQL_OPT_CAN_HANDLE_EXPIRED_PASSWORDS MYSQL_SERVER_PUBLIC_KEY                 MYSQL_TYPE_TIME                         mysql_fetch_fields                      mysql_stmt_bind_result
MYSQL_OPT_COMPRESS                     MYSQL_SET_CHARSET_DIR                   MYSQL_TYPE_TIMESTAMP                    mysql_fetch_row                         mysql_stmt_close
MYSQL_OPT_CONNECT_ATTR_ADD             MYSQL_SET_CHARSET_NAME                  MYSQL_TYPE_TINY                         mysql_field_count                       mysql_stmt_error
MYSQL_OPT_CONNECT_ATTR_DELETE          MYSQL_SET_CLIENT_IP                     MYSQL_TYPE_TINY_BLOB                    mysql_free_result                       mysql_stmt_execute
MYSQL_OPT_CONNECT_ATTR_RESET           MYSQL_SHARED_MEMORY_BASE_NAME           MYSQL_TYPE_VARCHAR                      mysql_get_ctype                         mysql_stmt_fetch
MYSQL_OPT_CONNECT_TIMEOUT              MYSQL_STMT                              MYSQL_TYPE_VAR_STRING                   mysql_get_field_types                   mysql_stmt_init
MYSQL_OPT_GUESS_CONNECTION             MYSQL_TIME                              MYSQL_TYPE_YEAR                         mysql_get_jtype_array                   mysql_stmt_num_rows
MYSQL_OPT_LOCAL_INFILE                 MYSQL_TIMESTAMP_DATE                    MySQLError                              mysql_get_julia_type                    mysql_stmt_prepare
MYSQL_OPT_NAMED_PIPE                   MYSQL_TIMESTAMP_DATETIME                MySQLHandle                             mysql_get_nullable                      mysql_stmt_result_metadata
MYSQL_OPT_PROTOCOL                     MYSQL_TIMESTAMP_TIME                    MySQLInterfaceError                     mysql_get_result_as_array               mysql_stmt_result_to_dataframe
MYSQL_OPT_READ_TIMEOUT                 MYSQL_TUPLES                            MySQLInternalError                      mysql_get_result_as_tuples              mysql_stmt_store_result
MYSQL_OPT_RECONNECT                    MYSQL_TYPE                              MySQLMetadata                           mysql_get_row_as_tuple                  mysql_store_result
MYSQL_OPT_SSL_CA                       MYSQL_TYPE_BIT                          MySQLResult                             mysql_get_row_as_vector                 mysql_subtype
MYSQL_OPT_SSL_CAPATH                   MYSQL_TYPE_BLOB                         MySQLRowIterator                        mysql_get_row_as_vector!                populate_row!
MYSQL_OPT_SSL_CERT                     MYSQL_TYPE_DATE                         MySQLStatementError                     mysql_init                              stmt_populate_row!
genie> MySQL.mysql_execute(conn, "SHOW COLUMNS from tweets")
ERROR: MethodError: Cannot `convert` an object of type Missings.Missing to an object of type String
This may have arisen from a call to the constructor String(...),
since type constructors fall back to convert methods.
Stacktrace:
 [1] setindex!(::Array{String,1}, ::Missings.Missing, ::Int64) at ./array.jl:583
 [2] insert_single_entry!(::DataFrames.DataFrame, ::Missings.Missing, ::Int64, ::Int64) at /Users/adrian/.julia/v0.6/DataFrames/src/dataframe/dataframe.jl:325
 [3] populate_row!(::DataFrames.DataFrame, ::Int64, ::Ptr{Ptr{Int8}}, ::Int64) at /Users/adrian/.julia/v0.6/MySQL/src/results.jl:195
 [4] mysql_result_to_dataframe(::MySQL.MySQLResult) at /Users/adrian/.julia/v0.6/MySQL/src/results.jl:210
 [5] #mysql_execute#17(::Int64, ::Function, ::MySQL.MySQLHandle, ::String) at /Users/adrian/.julia/v0.6/MySQL/src/handy.jl:160
 [6] mysql_execute(::MySQL.MySQLHandle, ::String) at /Users/adrian/.julia/v0.6/MySQL/src/handy.jl:142

genie> Pkg.status("MySQL")
 - MySQL                         0.3.0

genie> Pkg.status("DataFrames")
 - DataFrames                    0.11.2
```

**Comment by quinnj** [comment]
Sorry about the confusion. The new release is v0.4.0, which corresponds to the docs on the README. I guess this is a disadvantage of using the README for docs (lack of versioning). If you look at the git history for the README, you can see the old docs, or upgrade MySQL to v0.4.0 which has all the new functionality.

**Comment by essenciary** [comment]
Thank you - and Merry Christmas! :)

Yes, I found the old README but it's not useful anymore since `MySQL` 0.3.0 is broken on `DataFrames` 0.11.

I take it that 0.4.0 was not tagged yet - any plans on that anytime soon? 
```julia
genie> Pkg.available("MySQL")
genie> 5-element Array{VersionNumber,1}:
 v"0.0.1"
 v"0.0.2"
 v"0.1.0"
 v"0.2.0"
 v"0.3.0"
```

Also, any clue as to what's the deal with https://pkg.julialang.org? It still says the MySQL package was updated 5 days ago, so it looks like the time is not updated...
